### PR TITLE
Update to Bytes 0.5.2

### PR DESCRIPTION
**Pull request by anakos**
Hi-

Unsure if this contribution is welcomed, but here is a PR upgrading to the the 0.5.x series of the bytes library.  

Thanks
Alex

### CODE REVIEW

File: src/codec/length.rs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 use crate::{Decoder, Encoder};
-use bytes::{BigEndian, BufMut, ByteOrder, Bytes, BytesMut};
+use bytes::{Bytes, BytesMut, Buf, BufMut};
+use byteorder::{BigEndian, ByteOrder};
```
Comment (goto-bus-stop) [inline src/codec/length.rs]: it looks like this isn't used?

Comment (anakos) [reply src/codec/length.rs]: Sorry for just seeing this comment now! I am not sure what you mean here - what exactly is not being used?

Comment (goto-bus-stop) [reply src/codec/length.rs]: The `BigEndian` and `ByteOrder` imports from the `byteorder` crate—i think that entire dependency can be removed? :thinking:

Comment (anakos) [reply src/codec/length.rs]: Ok, I think I see what you mean. Pushing changes now - please let me know if this is what you had in mind. Thanks! Alex

**Comment by matthunz** [comment]
It's definitely welcome, thank you!

**Pull request merged**